README.md

```diff
@@ -187,6 +187,25 @@ With the help of [networkpolicy](https://kubernetes.io/docs/concepts/services-ne
 - Default deny networkpolicy is applied first
 - Then for each unique label in a namespace we have a networkpolicy with that label as a podSelector which allows traffic from pods which *don't* have some other randomly-selected label. This translates to 10 networkpolicies/namespace
 
+## EgressIP workloads
+
+This workload creates an egress IP for the client pods. SDN (OVN) will use egress IP for the traffic from client pods to external server instead of default node IP.
+
+Each iteration creates the following objects in each of the created namespaces:
+
+- 1 deployment with the configured number of client pod replicas. Client pod runs the https://github.com/martinkennelly/egress-ip-validator/pull/2 app which periodically sends http request to the configured "EXT_SERVER_HOST" server at an "DELAY_BETWEEN_REQ_SEC" interval with a request timeout of "REQ_TIMEOUT_SEC" seconds. Client pod then validates if the body of the response has configured "EGRESS_IPS". Once the client pod starts running and after receiving first succesful response with configured "EGRESS_IPS", it sets "eip_startup_latency_total" prometheus metric.
```

Comment:
Did you mean to link the PR directly instead of the repo?

Reply:
@smalleni currently we are using the image built from that PR. PR is not merged. I will request Martin to move this application to our redhat-performance org, but that may take sometime. This PR shouldn't be blocked because of these enhancements.

Reply:
Hi @venkataanil Dockerfile and its dependent contents can be moved to 
https://github.com/cloud-bulldozer/images - where other Containerfiles exist for all other apps.

Reply:
Done